src/OrleansServiceBus/Providers/EventDataGeneratorStreamProvider/EeventHubPartitionGeneratorReceiver.cs

```diff
@@ -47,7 +47,7 @@ public void ConfigureDataGeneratorForStream(IStreamIdentity streamId)
 
         public Task CloseAsync()
         {
-            return TaskDone.Done;
+            return Task.CompletedTask;
```

Comment:
not related to this change, but this file's name is a typo, I think: EeventHubXXX